agent/host_agent_suite_test.go

```diff
@@ -50,7 +50,6 @@ var (
 
 const (
 	bundleLookupBaseRegistry = "projects.registry.vmware.com/cluster_api_provider_bringyourownhost"
-	BundleLookupTag          = "v1.22.3"
 	K8sVersion               = "v1.22.3"
```

Comment:
@dharmjit Are we still at this version? Oh so only e2e we updated to v1.23.5 and integration tests use v1.22.3?

Reply:
We could update `k8s` version for agent tests as well.